src/Platform/TargetPhp/PhpBinaryPath.php

```diff
@@ -273,4 +273,23 @@ public static function fromCurrentProcess(): self
 
         return new self($phpExecutable, null);
     }
+
+    /**
+     * @param non-empty-string $minimumVersion
+     * @throws RuntimeException if version requirement not met
+     */
+    public function assertMinimumVersion(string $minimumVersion): void
```

Comment:
I don't think this is really needed, since the dependency resolution is already based on the target PHP version... What is the use case here? There aren't actually any usages of this new method...?

Reply:
You're absolutely right. I was thinking about runtime version validation but didn't consider that Composer's dependency resolution already handles this effectively through the `composer.json` requirements:

```json
{
    "require": {
        "php": ">=8.1"
    }
}
```

Since there's no immediate use case requiring runtime version validation beyond what Composer provides, I'll close this PR. Thank you for helping me avoid unnecessary code complexity!